common/lib/common-definitions/package.json

```diff
@@ -42,16 +89,16 @@
 		"lint:fix": "npm run prettier:fix && npm run eslint:fix",
 		"prettier": "prettier --check . --cache --ignore-path ../../../.prettierignore",
 		"prettier:fix": "prettier --write . --cache --ignore-path ../../../.prettierignore",
-		"tsc": "tsc",
+		"tsc": "fluid-tsc commonjs --project ./tsconfig.cjs.json && copyfiles -f ../../../common/build/build-common/src/cjs/package.json ./dist",
 		"tsc:watch": "tsc --watch",
```

Comment:
watch needs updated if going ESM first.
(I can push CJS first update if that is helpful. I wrote that quickly when I thought the build was failing per type test funkiness, but it is just the secret expiration.)

Reply:
I doubt the watch task is used, especially since this package is deprecated, so I'll just remove it.

Reply:
That's fine. Just so you know for other cases. We can use `fluid-tsc` to watch. The proper command (with CJS first) is
`copyfiles -f ../../../common/build/build-common/src/cjs/package.json ./dist && fluid-tsc commonjs --project ./tsconfig.cjs.json  --watch`